generate_overview_img.py

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Can you make this script executable? -> `chmod +x generate_overview_img.py`.

Reply:
Of course